pkg/service/repair/service.go

```diff
@@ -565,9 +566,15 @@ func (s *Service) hostRangeLimits(ctx context.Context, client *scyllaclient.Clie
 			return errors.Wrapf(err, "%s: get shard count", h)
 		}
 
+		// Since we don't issue repairs for per-shard ranges anymore 'intensity' becomes irrelevant
+		// Until we do a proper cleanup of 'intensity' related code let's restrict its ability
+		// to change the number of token ranges and allow only a single token range per host at a time
+		// as our documentation mandates.
 		v := rangesLimit{
-			Default: int(shards),
-			Max:     s.maxRepairRangesInParallel(shards, totalMemory),
+			Default: 1,
```

Comment:
Great. That was exactly my idea. We are defaulting to `1` which is both to preserve the documented behavior and to be on a conservative side by default - this can be seen as a "least noisy fastest mode (since `parallel` defaults to 0)".

And then we can change the intensity all the way up to Max (I tested that it works manually).